Nosach/web-quiz-project/src/main/java/servlet/QuizDeleteServlet.java

```diff
@@ -0,0 +1,27 @@
+package servlet;
+
+import service.QuizService;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class QuizDeleteServlet extends HttpServlet {
+
+    private QuizService ts = QuizService.QUIZ_SERVICE;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
```

Comment:
Better to do deleting via delete method of HTTP

Reply:
didn't find how to send delete request via html href or form without js. Is it needed?

Reply:
You can do this with form <form method="delete"...>, or with js.
With html href you can't